store/storage/rocksdb/db.go

```diff
@@ -146,15 +146,15 @@ func (db *Database) Get(storeKey string, version uint64, key []byte) ([]byte, er
 	return copyAndFreeSlice(slice), nil
 }
 
-// Prune attempts to prune all versions up to and including the provided version.
+// LazyPrune attempts to prune versions up to and including the given version.
 // This is done internally by updating the full_history_ts_low RocksDB value on
 // the column families, s.t. all versions less than full_history_ts_low will be
 // dropped.
 //
 // Note, this does NOT incur an immediate full compaction, i.e. this performs a
 // lazy prune. Future compactions will honor the increased full_history_ts_low
 // and trim history when possible.
-func (db *Database) Prune(version uint64) error {
+func (db *Database) LazyPrune(version uint64) error {
```

Comment:
@alexanderbez , the test is failed due to the lazy mode. I've added the instant pruning, wdyt?

Reply:
btw, I don't think we need `LazyPrune`